pkg/ui/frontend/src/components/NAS/Monitor/StepInfo.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+
+import * as d3             from 'd3'
+import * as d3Graphviz     from 'd3-graphviz'
+
+
+const styles = theme => ({
+})
+
+class StepInfo extends React.Component {
+
+    componentDidMount() {
+
+        const id = `graph${this.props.id}`
+        d3.select(`#${id}`)
+        .graphviz()
+        .renderDot(this.props.step.architecture)
+        .width(640)
+        .height(480)
+        .fit(true)
+    }
+
+    render() {
+
+        const { step, classes } = this.props;
+        const id = `graph${this.props.id}`
+        return (
+            <div className={classes.root}>
+                <Typography variant={"h6"}>
+                    Architecture
+                </Typography>
+                <div id={id} style={{textAlign: "center !important", margin: '0 auto'}} />
```

Comment:
this inline style can be pulled out into the style definition. Also, instead of using `!important` if there's a way to use multiple stringed class declaration, I'd use that. css in js is relatively new (in alpha actually), so not sure if it's possible so if not then `!important` is fine.

Reply:
Fixed.